### ISSUE TITLE
Article breadcrumbs use aria-current but do not point to the current page

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#6296]
The article breadcrumbs template uses `aria-current="page"` on the last item in the breadcrumb. However, the breadcrumbs are designed to only include the article section, not the current article title. This is confusing because `aria-current="page"` is supposed to identify the _current page_, not the category or section that this page is a part of.

However, we don't want to include the full article title in the breadcrumb, because it would make the breadcrumb very long and would repeat the article title. Instead, the article title should be added as the last item of the breadcrumb, but that breadcrumb should be hidden from all but screen readers.

Example:

```html
<nav role="navigation" aria-label="You are here:">
	<ol>
		<li>
			Home
		</li>
		<li>
			Archives
		</li>
		<li>
			Vol 1. No 1 (2019)
		</li>
		<li class="current pkp_screen_reader">
			<span aria-current="page">
				Signalling Theory Dividends: A Review Of The Literature And Empirical Evidence
			</span>
		</li>
	</ol>
</nav>
```

**Comment by ctgraham** [comment]
If we hide the "current page" breadcrumb for articles, this leaves incongruity with the other pages which do have a "current page" breadcrumb.  In almost all other cases (Archives, Submissions, Editorial Team, etc), the "current page" breadcrumb is the only breadcrumb other than "HOME", so hiding the "current page" breadcrumb uniformly seems problematic.

I'd suggest that the current-page breadcrumb on the article landing page read "Abstract".  Then, if subpages for the article were introduced, the breadcrumb "Abstract" could return to the landing page, and additional subpages (like galleys) could link to the "Abstract" as a breadcrumb.

**Comment by NateWr** [comment]
The breadcrumb template for articles and the breadcrumbe template for other pages are different, so you can only hide the current page breadcrumb on the article page without modifying the others. I'm not too worried about there being a visual difference between the pages. 

> I'd suggest that the current-page breadcrumb on the article landing page read "Abstract".

The problem is that a lot of journals do not publish abstracts. And our goal is to have the full article text on the landing page.

**Comment by ctgraham** [comment]
How about just "Article" as the breadcrumb then?  This would allow a consistent, short, but meaningful slug to exist for cases where we want to place another breadcrumb under the Article landing page.

**Comment by NateWr** [comment]
The trouble is that not everything is an "Article". It may be a review or editorial, for example. That is kind of the idea behind using the [section name](https://github.com/pkp/ojs/blob/master/templates/frontend/pages/article.tpl#L24).

We do have a field in the section settings for "Identify items published in this section as a(n)" (that's `identifyType`). Maybe we can use that with a fallback to the article title?

I'm not sure if this would satisfy the requirements for `aria-current="page"`. @israelcefrin what do you think if the breadcrumb was like this:

```
Home / Vol 1. No 1. / Article
```

Would that be descriptive enough for `aria-current="page"` or do we need to use the article title?

**Comment by israelcefrin** [comment]
Reviewing this still open issue, to make it more accessible, we should replace "article" by the article title. Otherwise, it may mislead users to think that "Article" is the title itself. 
Also, I've noticed that we are using the aria attribute r`ole="navigation"` on a `nav` html element. The attribute is not required here, since it is meant to be used on menus that rely on legacy narkup and navigational components that don't use the `nav` tag.
